lib/src/user_location_layer.dart

```diff
@@ -113,28 +124,53 @@ class _MapsPluginLayerState extends State<MapsPluginLayer>
           }
           //widget.options.markers.clear();
 
+          print("Direction : "+(_direction ?? 0).toString());
```

Comment:
Just above it we can initialize the direction as:

`_direction = onValue.heading`